src/_data/side-nav.yml

```
@@ -49,6 +49,8 @@
       permalink: /guides/language/extension-methods
     - title: Numbers
       permalink: /guides/language/numbers
+    - title: Concurrency
+      permalink: /guides/language/concurrency
```

(Optionally), what do you think about flipping the order of these? I feel the concurrency discussion is generally more useful and it seems to look a bit better.

```suggestion
    - title: Concurrency
      permalink: /guides/language/concurrency
    - title: Numbers
      permalink: /guides/language/numbers
```

---

sgtm, done~